src/Microsoft.DotNet.Wpf/test/System.Xaml/Common/CustomXamlNameResolver.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Xaml.Tests.Common
```

Comment:
what makes these "Common"?

Reply:
oh i see, they are test helper classes, i thought these were tests 😂

Reply:
What are your thoughts on using something like the Moq framework (like they use in winforms)

Would remove the need for this type of CustomXYZ class!

Reply:
let me go take a look at how it's used in WinForms. just read the github page and it looks awesome!

Reply:
I think I'm starting to understand how it's used, and that does look pretty nice! I think it'll be pretty clear if you're able to make a small change and push an update? Don't change everything to it just in case, but it looks fairly righteous

Reply:
You'll need to push another change anyways to kick off another PR and CI run so we can see if the tests are running anyways :)